Reference/conformance/conformance.pb.swift

```diff
@@ -154,6 +157,9 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message {
     }
   }
 
+  /// Used by the SwiftProtobuf encoding machinery, not generally used directly.
+  /// `Message.serializedData()`, `Message.jsonUTF8Data()`, and other serializer methods
+  /// are defined in `Message` and `Message+*Additions` files.
```

Comment:
As above, we should be clear that these "Message and Message+Additions files" are located in the SwiftProtobuf library source code.